kodilla-spring-basic/src/main/java/com/kodilla/spring/basic/spring_configuration/homework/Cabrio.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.spring.basic.spring_configuration.homework;
+
+public class Cabrio implements Car {
```

Comment:
dodaj klase abstrakcyjna pomiedzy interfejsem Car a konkretnym samochodem, np. public abstract class AbstractCar implements Car a w niej pole private boolean headlightsTurnedOn oraz konstruktor public AbstractCar(int hour) { if (time >= 20 || time < 6) this.headlightsTurnedOn = true; else  this.headlightsTurnedOn = false; }i metode hasHeadlightsTurnedOn() { return headlightsTurnedOn; }